packages/components/src/components/button-tile/__snapshots__/button-tile-styles.spec.ts.snap

```diff
@@ -269,21 +270,22 @@ p {
   position: absolute;
   overflow: hidden;
   border-radius: 8px;
+  transform: translateZ(0);
```

Comment:
What is this good for? Nothing is `position: fixed` in this component.

Reply:
This fixes a bug in safari where the corners are not rounded on hover. Comment is next to the implementation.